examples/jiralert.yml

```diff
@@ -25,6 +25,8 @@ defaults:
   reopen_duration: 0h
   # Static label that will be added to the JIRA ticket alongisde the JIRALERT{...} or ALERT{...} label
   static_labels: ["custom"]
+  # Other projects that issues may be moved to and further updates are desired
```

Comment:
```suggestion
  # Other projects are the projects to search for existing issues for the given alerts if
  # the main project does not have it. If no issue was found in, the main projects will
  # be used to create a new one. If the old issue is found in one of the other projects
  # (first found is used in case of duplicates) that old project's issue will be used for
  # alert updates instead of creating on in the main project.
```